Update WMI check to be officially legacy/unsupported (#16064)
DataDog/integrations-core: wmi_check/README.md

## Overview

> **Note:** This integration is no longer maintained or recommended for use. Please use the [Windows performance counters integration][2] instead in all cases due to its significantly lower overhead and thus better scalability.

Reviewer: ```suggestion
**Note:** Datadog no longer maintains or recommends the use of this integration. Instead, use the [Windows performance counters integration][2] in all cases due to its significantly lower overhead and thus better scalability.
```
